flipt-client-csharp/src/FliptClient/Models/Response.cs

```diff
@@ -0,0 +1,107 @@
+using System.Text.Json.Serialization;
+
+namespace FliptClient.Models
+{
+    public class Flag
+    {
+        [JsonPropertyName("key")]
+        public string Key { get; set; }
+
+        [JsonPropertyName("enabled")]
+        public bool Enabled { get; set; }
+
+        [JsonPropertyName("type")]
+        public string Type { get; set; }
+    }
+
+    public class VariantEvaluationResponse
+    {
+        [JsonPropertyName("flag_key")]
+        public string FlagKey { get; set; }
+
+        [JsonPropertyName("match")]
+        public bool Match { get; set; }
+
+        [JsonPropertyName("segment_keys")]
+        public List<string> SegmentKeys { get; set; }
```

Comment:
i should prob make this a `string[]` to be consistent with other collections elsewere in this file, as we don't really need it to be a List as its size isn't changed after evaluation